docs/reference/modules/cluster/disk_allocator.asciidoc

```diff
@@ -15,29 +15,32 @@ file or updated dynamically on a live cluster with the
 
 `cluster.routing.allocation.disk.watermark.low`::
 
-    Controls the low watermark for disk usage. It defaults to 85%, meaning ES will
-    not allocate new shards to nodes once they have more than 85% disk used. It
-    can also be set to an absolute byte value (like 500mb) to prevent ES from
-    allocating shards if less than the configured amount of space is available.
+    Controls the low watermark for disk usage. It defaults to `85%`, meaning
+    that Elasticsearch will not allocate shards to nodes that have more than
+    85% disk used. It can also be set to an absolute byte value (like `500mb`)
+    to prevent Elasticsearch from allocating shards if less than the specified
+    amount of space is available. This setting has no effect on brand-new
+    shards that have never been allocated before.
```

Comment:
To make this a little bit more explicit, I would add something along the lines of ", for example when a new index is created".